simple-facebook-instant-articles.php

```diff
@@ -74,6 +78,34 @@ public function __construct( $file, $version ) {
 		$this->template_path = trailingslashit( $this->dir ) . 'templates/';
 		$this->home_url      = trailingslashit( home_url() );
 		$this->endpoint      = apply_filters( 'simple_fb_article_endpoint', 'fb-instant' );
+		$this->options       = get_option( 'fb_instant' );
+	}
+
+	/**
+	 * The register activation hook should flush the rewrite rules, in the event
+	 * that it doesn't let's go ahead and flush the rules.
+	 *
+	 * @return void
+	 */
+	public function flush_rules() {
```

Comment:
I've asked around and was told that flushing isn't working in core particularly well: https://core.trac.wordpress.org/ticket/30501

It needs to be done on activating/deactivating the plugin, plugin updates that change the registered rules, stuff like that.